index.markdown

```diff
@@ -4,6 +4,10 @@
 
 layout: events_layout
 ---
+{% include edit_on_github.html %}
```

Comment:
this should not be needed since it's already included in the `layout: events_layout`